app/lib/.server/llm/model.ts

```diff
@@ -56,6 +62,25 @@ export function getGroqModel(apiKey: string, model: string) {
   return openai(model);
 }
 
+export function getAmazonBedrockModel(apiKey: string, model: string) {
```

Comment:
Seems like we will need to add more to add support for this, as it requires 3 parameters instead of one API key.

Reply:
I had a test run with this, and it seems to be working. If you wanna see just lmk.

Reply:
What I mean is that its not possible to set API key from UI right?
![image](https://github.com/user-attachments/assets/bfbff822-7899-4c07-8c99-1202780fae95)

It only works when set locally in env parameters